custom_addons/softifi_graph_module_dependency/static/src/components/module_graph/GraphModuleComponent.js

```
@@ -0,0 +1,639 @@
+  get selectedModuleObjects() {
+    return Array.from(this.state.selectedModules)
+      .map((id) => this.state.nodes.find((node) => node.id === id))
+      .filter(Boolean);
+  }
+
+  /**
+   * Remove a module from selected modules and graph
+   */
+  removeSelectedModule(moduleId) {
+    this.state.selectedModules.delete(moduleId);
+    if (this.graphNodes && this.graphNodes._data[moduleId]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)